datafusion/physical-expr/src/expressions/column.rs

```diff
@@ -107,6 +102,18 @@ impl PhysicalExpr for Column {
     ) -> Result<Arc<dyn PhysicalExpr>> {
         Ok(self)
     }
+
+    /// Return the boundaries of this column, if known.
+    fn analyze(&self, context: &mut AnalysisContext) -> Option<ExprBoundaries> {
```

Comment:
Why not `boundaries` or `expression_boundaries`

Reply:
I think having `boundaries()` and `apply_boundaries` makes it consistent. Will change, thanks!